setup.py

```diff
@@ -1,32 +1,49 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+"""
+Flask-Whooshee
+--------------
 
-try:
-    from setuptools import setup
-except:
-    from distutils.core import setup
+Adds Whoosh support to Flask-SQLAlchemy.
 
+Links
+`````
+
+* `Documentation <https://pythonhosted.org/Flask-Whooshee/>`_
```

Comment:
I have to say I'd prefer readthedocs.io - pythonhosted doesn't allow automatic documentation builds on every push AFAICT. I registered the url for flask-whooshee right now, so let's use it: http://flask-whooshee.readthedocs.io/en/latest/